sys/include/net/gnrc/gomach/gomach.h

```diff
@@ -7,10 +7,25 @@
  */
 
 /**
- * @defgroup    net_gnrc_gomach A traffic-adaptive multi-channel MAC
+ * @defgroup    net_gnrc_gomach GoMacH
  * @ingroup     net_gnrc
- * @brief       A traffic adaptive MAC protocol that provides high traffic
- *              adaptability, high energy efficiency and high robustness.
+ * @brief       GoMacH is, we believe, "a General, nearly Optimal MAC protocol
```

Comment:
Do we believie it or know it. Anyway this `, we believe,` should be removed IMHO.